README.md

```diff
@@ -218,6 +218,40 @@ $ make test-unit
 $ make test-integration
 ```
 
+### Working with recipes
+
+#### Core recipe library
+
+A core library of installation recipes is included with the CLI for use within the
+`install` command. Recipe files are syndicated from [open-install-library](https://github.com/newrelic/open-install-library)
+and embedded in the CLI binary at release time. To fetch the latest recipe library
+while developing, the following make target can be used:
+
+```
+make recipes
+```
+
+Recipe files are stored in `internal/install/recipes/files`. Once files have been
+fetched, they will be included in future CLI builds. If a particular version of
+the recipe library is desired, the archive download URL can be passed to the make
+target via the `RECIPES_ARCHIVE_URL` option:
+
+```
+make recipes RECIPES_ARCHIVE_URL=https://github.com/newrelic/open-install-library/releases/download/v0.49.28/recipes.zip
```

Comment:
maybe update the version number to our first actual true release of the library. @ctrombley

Reply:
👍